client/src/locales/en.json

```diff
@@ -216,6 +216,11 @@
   "dashboard.index.no-arch-sacks": "There are no archived sacks yet.",
   "dashboard.index.no-sacks": "There are no sacks yet.",
   "dashboard.index.show-arch-sacks": "Show archived sacks",
+  "delete-link.has-expired-part1": "Delete link has expired. Please go to",
+  "delete-link.has-expired-link": "user profile page ",
+  "delete-link.has-expired-part2": "and find delete account option to get new delete link.",
+  "email.delete-account.failure": "There was an error while sending email.",
+  "email.delete-account.success": "Email with delete button was sent to your email inbox. To delete your account follow email instructions.",
```

Comment:
Check other services what text is used. Most probably sth like "Delete account confirmation email was sent...". Please check and use more accurate text without some 'button' text.